lib/services/rating_calculators/for_weekly_votes_per_day.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Services::RatingCalculators
+  class ForWeeklyVotesPerDay
+    MAX_COUNTED_VOTES_PER_WEEK = 100.0
+    private_constant :MAX_COUNTED_VOTES_PER_WEEK
+    attr_reader :votes_for_this_server
+
+    def initialize(votes_for_this_server)
+      @votes_for_this_server = votes_for_this_server
+      @votes_for_last_week = votes_for_last_week
```

Comment:
looks like might be better to store average_votes_per_day instead, because we use this value in formulas